test-fixture.html

```diff
@@ -309,8 +309,12 @@
       );
     });
 
-  document.registerElement('test-fixture', {
-    prototype: TestFixturePrototype
-  });
+  try {
+    document.registerElement('test-fixture', {
+      prototype: TestFixturePrototype
+    });
+  } catch (e) {
+    console.warn('test-fixture has already been registered!');
```

Comment:
Maybe it would be helpful to add, "if you are using WCT, you do not need to manually import `test-fixture.html" or something to that effect. What do you think?